dev-tools/packaging/packages.yml

```diff
@@ -45,6 +45,10 @@ shared:
         source: 'elastic-agent.yml'
         mode: 0600
         config: true
+      /etc/{{.BeatName}}/capabilities.yml:
```

Comment:
this needs to be added in multiple places. 
try searching for `'elastic-agent.yml'` there should be 5 occurences, you have 1 so far.